iamproperty/iamproperty.github.io
# Basic form input

## Summary of Changes:
1. Vue component
2. New and modified CSS
3. No JS changes

## Rationale
Have consistent and usable input fields. Remove the need for javascript plugins for datepickers etc.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /elements/form

## Ticket(s)
DS-40

## Checklist

## Notes
I have updated the threshold for error on the visual regression to reduce the number of failures. Its still a really low threshold but will not ignore really minor issues due to rendering differences.
